server/Services/Chat/Chat.Domain/ChatRoom.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Chat.Domain;
+
+public class ChatRoom
+{
+    public Guid Id { get; set; }
+    public Guid RecruiterId { get; set; }
+    public Guid ApplicantId { get; set; }
```

Comment:
Note: domain models were changed. See ITF.Domain/Entities.

Employer -> Recruiter
Applicant -> Developer

Reply:
Also... Will it be a good idea to tie chat implementation to recruiters/developers?

Reply:
Will chat be used for recruiters/recruiters or developers/developers?